api/src/main/java/io/minio/DateFormat.java

```diff
@@ -38,7 +38,7 @@
       DateTimeFormat.forPattern("yyyyMMdd").withZoneUTC().withLocale(Locale.US);
 
   public static final DateTimeFormatter HTTP_HEADER_DATE_FORMAT =
-      DateTimeFormat.forPattern("EEE',' dd MMM yyyy HH':'mm':'ss zzz").withZoneUTC().withLocale(Locale.US);
+      DateTimeFormat.forPattern("EEE',' dd MMM yyyy HH':'mm':'ss 'GMT'").withZoneUTC().withLocale(Locale.US);
```

Comment:
It is not the same 

Without 
```
x-amz-copy-source-if-unmodified-since: Fri, 03 Apr 2015 17:10:00 +00:00
``` 

With
```
x-amz-copy-source-if-unmodified-since: Fri, 03 Apr 2015 17:10:00 GMT
```